build/configs/rtl8730e/loadable_ext_psram/defconfig

```diff
@@ -1031,6 +1031,12 @@ CONFIG_LWNL80211=y
 # CONFIG_DEBUG_LWNL80211_VENDOR_DRV_ERROR is not set
 CONFIG_NET_LOOPBACK_INTERFACE=y
 
+# WIFI CSI Support
```

Comment:
DO NOT modify defconfig manually.
After doing through menuconfig, and then copy changes into the defconfig.

Reply:
Done